udemy_exercises/songs/Main.java

```diff
@@ -0,0 +1,175 @@
+
+import java.util.*;
+
+public class Main {
+
+    private static ArrayList<Album> albums = new ArrayList<Album>();
+
+    public static void main(String[] args) {
+
+        Album album = new Album("Clarity", "Zedd");
+        album.addSong("Stay", 3.30);
+        album.addSong("Find You", 3.24);
+        album.addSong("Clarity", 4.32);
+
+        albums.add(album);
+
+        album = new Album("Speak Those Things", "Fred Hammond");
+        album.addSong("You Are My Daily Bread", 4.37);
+        album.addSong("Lord of the Harvest", 5.20);
+        album.addSong("Show Yourself Strong", 4.54);
+        album.addSong("Everything to Me", 3.53);
+        album.addSong("Great", 4.51);
+        album.addSong("That Ain't Nothin'", 5.44);
+        albums.add(album);
+
+        LinkedList<Song> playList = new LinkedList<Song>();
+        albums.get(0).addToPlayList("Find You", playList);
+        albums.get(0).addToPlayList("Great", playList);
+        albums.get(0).addToPlayList("American Idiot", playList);
+        albums.get(0).addToPlayList(3, playList);
+        albums.get(1).addToPlayList(1, playList);
+        albums.get(1).addToPlayList(2, playList);
+        albums.get(1).addToPlayList(4, playList);
+        albums.get(1).addToPlayList(32, playList);
+
+        play(playList);
+
+
+
+
+    }
+
+    private static void play(LinkedList<Song> playList) {
+        Scanner scanner = new Scanner(System.in);
+        boolean quit = false;
+        boolean forward = true;
+        ListIterator<Song> listIterator = playList.listIterator();
+        if(playList.size() == 0) {
```

Comment:
What led to the decision to require a populated playlist before 'play' functionality is available?  How might this method change to allow at least 'display menu' and 'add song' functionality?